src/module.c

```diff
@@ -5587,6 +5596,17 @@ robj **moduleCreateArgvFromUserFormat(const char *cmdname, const char *fmt, int
  *              same as the client attached to the given RedisModuleCtx. This will
  *              probably used when you want to pass the reply directly to the client.
  *     * `C` -- Check if command can be executed according to ACL rules.
+ *     * 'S' -- Run the command in a script mode, this means that it will raise an error
+ *              if a command which are not allowed inside a script (flagged with the no-script flag)
+ *              is invoked (like shutdown).
+ *              In addition, on script mode, write commands are not allowed if there is
+ *              not enough good replicas (as configured with repl_min_slaves_to_write)
+ *              and/or a disk error happened.
```

Comment:
```suggestion
 *     * 'S' -- Run the command in a script mode, this means that it will raise an error
 *              if a command which are not allowed inside a script (flagged with the no-script flag)
 *              is invoked (like SHUTDOWN).
 *              In addition, on script mode, write commands are not allowed if there are
 *              not enough good replicas (as configured with repl_min_slaves_to_write)
 *              or when the server is unable to persist to the disk.
```